app/chat/layout.tsx

```
@@ -65,49 +66,51 @@ import useSWRInfinite from 'swr/infinite';
+                    revalidateSessions: () => swrMutate(undefined, { revalidate: true })
+                }}>
+                    <div className="flex-1 flex flex-col min-w-0">
+                        {/* Non-blocking error banner */}
+                        {error && (
+                            <div className="absolute top-2 right-2 z-50 bg-red-500/20 border border-red-500/40 text-red-200 text-sm px-3 py-2 rounded-md backdrop-blur">
+                                <div className="flex items-center gap-3">
+                                    <span>Failed to load sessions</span>
+                                    <button
+                                        onClick={() => setSize(1)}
+                                        className="px-2 py-0.5 text-xs rounded bg-red-500/30 hover:bg-red-500/40 border border-red-500/50"
+                                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)